test/e2e/common/configmap.go

```diff
@@ -133,6 +133,24 @@ var _ = Describe("[sig-node] ConfigMap", func() {
 		configMap, err := newConfigMapWithEmptyKey(f)
 		framework.ExpectError(err, "created configMap %q with empty key in namespace %q", configMap.Name, f.Namespace.Name)
 	})
+
+	/*
+	   Release : v1.16
+	   Testname: Update ConfigMap
+	   Description: Attempt to update a ConfigMap after create one.
+	*/
+	framework.ConformanceIt("should patch ConfigMap successfully", func() {
+		name := "configmap-test-" + string(uuid.NewUUID())
+		configMap := newConfigMap(f, name)
+		ginkgo.By(fmt.Sprintf("Creating configMap %v/%v", f.Namespace.Name, configMap.Name))
+
+		configMap.Data = map[string]string{
+			"data": "value",
+		}
+		ginkgo.By(fmt.Sprintf("Updating configMap %v/%v", f.Namespace.Name, configMap.Name))
+		_, err := f.ClientSet.CoreV1().ConfigMaps(f.Namespace.Name).Create(configMap)
```

Comment:
This neither updates nor patches this configmap...

Reply:
Sorry for the careless 😢 Just updated the PR, PTAL